File: src/ERP/acme.atena.domain/DTO/Person/Pessoa.cs
using acme.atena.domain.DTO.Account;
using acme.atena.domain.DTO.Product;
using acme.atena.domain.DTO.Seguranca;
using System;
using System.Collections.Generic;
using System.Text;

namespace acme.atena.domain.DTO.Person
{
    public abstract class Pessoa : AbstractEntity
    {
        public Pessoa()
        {
            Dividas = new HashSet<Divida>();
            Pagamentos = new HashSet<Pagamento>();
        }

        public string Nome { get; set; }
        public string CpfCnpj { get; set; }
        public string Email { get; set; }
        public string Celular { get; set; }
        public DateTime DataNascimento { get; set; }

        public virtual ICollection<Pagamento> Pagamentos{ get; set; }
        public virtual ICollection<Divida> Dividas { get; set; }
        

    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/person/pessoa.ts
import { Divida } from './../account/divida';
import { Pagamento } from './../account/pagamento';
import { AbstractEntity } from '../abstract-entity';
import { UsuarioResponse } from '../security/usuario';

export abstract class Pessoa extends AbstractEntity{
  nome: string = "";
  cpfCnpj: string = "";
  email: string = "";
  celular: string = "";
  dataNascimento: Date = new Date();

  pagamentos: Pagamento[]= [];
  dividas : Divida[]= [];
}
